shared/wallets/wallet/settings/index.stories.tsx

```diff
@@ -64,11 +64,14 @@ const sharedSettingsProps = {
   onCurrencyChange: Sb.action('onCurrencyChange'),
   onDelete: Sb.action('onDelete'),
   onEditName: Sb.action('onEditName'),
+  onLoadSecretKey: Sb.action('onLoadSecretKey'),
   onMobileOnlyModeChange: Sb.action('onMobileOnlyModeChange'),
+  onSecretKeySeen: Sb.action('onSecretKeySeen'),
   onSetDefault: Sb.action('setDefault'),
   onSetupInflation: Sb.action('onSetupInflation'),
   refresh: () => {},
   saveCurrencyWaiting: false,
+  secretKey: '',
```

Comment:
It may be useful to set the secret key for at least one story, right now every settings story displays the loading message.